spec.md

```diff
@@ -2301,6 +2301,9 @@ If plugin has `STAGE_UNSTAGE_VOLUME` node capability then:
 
 Otherwise `NodeExpandVolume` MUST be called after successful `NodePublishVolume`.
 
+If a plugin only has `VolumeExpansion.OFFLINE` capability then it MUST be first expanded using `ControllerExpandVolume` by taking the volume offline(see `ControllerExpandVolume` for more details) and then node-staged or node-published before it can be expanded on the node via `NodeExpandVolume`.
```

Comment:
/by taking the volume offline/d -- I don't think this reads well.

Suggest something like:
If a plugin only supports expansion via the `VolumeExpansion.OFFLINE` capability, then the volume MUST first be taken offline and expanded via `ControllerExpandVolume` (see ... for more details), and then ...

Some nits:
- missing space between word and paren: "offline("
- nix the extra newline that was added after the new sentence (only need one blank line between the sentence and the proto spec code block)